java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -3376,16 +3376,27 @@ void testFixedWidthCast() {
   void testCastBigDecimalToString() {
     BigDecimal[] bigValues = {new BigDecimal("923121331938210123.321"),
         new BigDecimal("9223372036854775808.191"),
+        new BigDecimal("-9.223"),
```

Comment:
Should we add a test case for a negative decimal value greater than 64 bits?

Reply:
The only thing that will accomplish is to verify that Java BigDecimals > 64 bit have precision > 38